docs/sg_execution_times.rst

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
This file should be removed from this PR. (It should be added to .gitignore -- I can do that in a separate PR, or you can add it to this one.)

Reply:
no problem, will do it now, thank you